management-api-common/src/main/java/com/datastax/mgmtapi/shims/CassandraAPI.java

```diff
@@ -66,4 +66,6 @@ default Object handleRpcResult(Callable<Object> rpcResult) throws Exception
     RpcStatementShim makeRpcStatement(String method, String[] params);
 
     HintsService getHintsService();
+
+    List<String> getKeyspaces();
```

Comment:
Since the implementation for all 3 of the "shims" is the same, we could simple implement a `default` here in the interface, then the implementations wouldn't have to override anything.

Reply:
Cool trick, I didn't know there was such a thing as a `default` implementation in Java interfaces 😅 
Always thought that's what abstract classes were for...
Thanks!

Reply:
Yep, it's "new" as of Java 8. The use case for it is to be able to introduce new APIs without breaking older implementations that don't implement them.